### ISSUE TITLE
"Matching Recents Capsule" breaks scrolling 

### ISSUE THREAD

**Issue by Hope-Upstairs** [SteamGridDB/decky-steamgriddb#127]
If I enable "Matching Recents Capsule", whenever I scroll (either with the touchscreen or shift+scroll on mouse&keyboard), the row goes back to whichever entry was highlighted (with the dpad/joystick/arrow keys).

**Comment by cheatfreak47** [comment]
Confirming this issue. It's quite confusing when you discover it. Hope this one can be fixed at some point.

**Comment by SanktyRG** [comment]
Hello, I came here because I have the same or at least a pretty similar issue, when using "Matching Recents Capsule" or "Square Capsules", I can no longer change Home tabs (What's New, Friends and Recommended)

**Comment by cheatfreak47** [comment]
Yeah, this is a known issue, I'm surprised it hasn't been looked into yet, considering it breaks core menu functionality, but, hey it do be like that sometimes.
